INSTALL.md

```diff
@@ -47,15 +58,15 @@
 
 ## Linux
 
-  - First, try your package manager.
-    Pandoc is in the [Debian], [Ubuntu], [Slackware],
+  - Older versions of pandoc can be installed using most package
```

Comment:
It's not strictly accurate to say that the package managers have older versions.
Arch and NiXOS, for example, are usually pretty current.  (gentoo too, I imagine.)
It depends also on the rate of release of pandoc.